snyk-monitor/values.yaml

```diff
@@ -8,10 +8,17 @@ monitorSecrets: snyk-monitor
 certsConfigMap: snyk-monitor-certs
 registriesConfConfigMap: snyk-monitor-registries-conf
 
-# The ConfigMap to use for loading policies into snyk-monitor.
-workloadPoliciesMap: snyk-monitor-workload-policies
-# A list of Snyk Organization public IDs to let snyk-monitor know in which Organization to auto-import and auto-delete scanned images.
-policyOrgs:
+# An external ConfigMap to use for loading policies into snyk-monitor.
+# When using this the default policy shouldn't be deployed, set `defaultWorkloadPolicy.enabled: false`.
+workloadPoliciesMap: ''
+
+# Default rego workload policies to install.
+# When setting `workloadPoliciesMap`, this should be disabled.
+defaultWorkloadPolicies:
+  enabled: true
+  name: snyk-monitor-workload-policies
+  # A list of Snyk Organization public IDs to let snyk-monitor know in which Organization to auto-import and auto-delete scanned images.
+  policyOrgs: []
```

Comment:
Could you please also add support for policyOrgs to be defined in the global scope, so that this change will not break current functionality?
This will also require a change in the configmap, so that we have a fallback to the global policyOrgs in case the defaultWorkloadPolicies.policyOrgs is not defined.

Reply:
Is the idea of not changing the current api? (I consider values.yaml as an api). If we are keeping policyOrgs in global scope and the enabled key in default policies can be removed, the there's probably better to just set `defaultWorkloadPoliciesMap` to the name of the default policy and verify if WorkloadPoliciesMap is defined to override that. What do you think?

Reply:
Sounds like a good idea 👍